src/playwright-runner.js

```diff
@@ -57,7 +57,7 @@ async function createJob (suiteName, hasPassed, startTime, endTime, args, playwr
     }
   }
 
-  generateJunitfile(cwd, suiteName, args.param.browserName);
+  generateJunitfile(cwd, suiteName, args.param.browser);
```

Comment:
Hm. Confused by this change due to https://github.com/saucelabs/saucectl/blob/main/internal/playwright/config.go#L70

Reply:
`browserName` doesn't work. It has been moved to `browser` in this method https://github.com/saucelabs/sauce-playwright-runner/blob/master/src/utils.js#L153